Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,49 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
```

Comment:
Shouldn't the special treatment of lists and strings with newlines be mentioned somewhere here?

Reply:
It depends.. if you need to parse some log file, generated with this mechanism -- will you look into the example file to decide what you need to do, or go to the logging implementation?

Reply:
I agree that when tasked with parsing the log the person in question would be more likely to check the examples, but wouldn't it be easier to use the function for its intended purpose if this treatment is mentioned? Someone may assume that function is meant for single lines only and proceed to explicitly separate strings and put them into separate `log()` calls.

Reply:
Added more detailed description: 273a39f.